joomla/joomla-cms
# [5.3] Fix author not being updated in Tagged Item upon saving article

Currently, when saving an article, the `core_created_user_id` and `core_created_by_alias` fields are not updated in the corresponding entry in the `#__ucm_content` table.

### Summary of Changes

Add both fields to the fields_mapping column in the `#__content_types` table for articles.

### Testing Instructions

A test should be to make a new installation with this PR to see if the changes in the base.sql are right, and another test should be made to update an existing 5.3.x to the patched package created by Drone for this PR to see if the update SQL script is run and has no errors.

After that (new installation or update), change the author (Created By) of an article. Take a look at the corresponding entry in the `#__ucm_content` table.

### Actual result BEFORE applying this Pull Request

`core_created_user_id` and `core_created_by_alias` are not updated.

### Expected result AFTER applying this Pull Request

`core_created_user_id` and `core_created_by_alias` are updated.

### Link to documentations
Please select:
